Sources/NIOCore/ByteBuffer-int.swift

```diff
@@ -113,6 +113,22 @@ extension ByteBuffer {
             self.setBytes(ptr, at: index)
         }
     }
+
+    /// Peeks at the integer at the current reader index without advancing it.
```

Comment:
```suggestion
    /// Returns the integer at the current reader index without advancing it.
```

Reply:
Just to echo my comment on one of the issues referred to in the description. Since this kind of a change can be extrapolated to every type that has a get* API, can the scope of this PR increase to consider them or should they separate PRs?
Thanks!

Reply:
Given that this PR is effectively good to go I'd bias towards merging this and opening a separate PR for other methods.